Upgrade CAPI commands if from 1.8 to 2.1 first (#4424)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md

If you are running on more than one management cluster (Kommander cluster), you must upgrade the CAPI components on each of these clusters.

<p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster where you want to run the upgrade by setting the <code>KUBECONFIG</code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
    

> 1.  If your cluster was upgraded to 2.1 from 1.8, prepare the old cert-manager installation for removal:

Reviewer: ```suggestion
1.  If your cluster was upgraded to 2.1 from 1.8, prepare the old cert-manager installation for upgrade:
```

    ```bash
    helm -n cert-manager get manifest cert-manager-kubeaddons | kubectl label -f - clusterctl.cluster.x-k8s.io/core=cert-manager
    kubectl delete validatingwebhookconfigurations/cert-manager-kubeaddons-webhook mutatingwebhookconfigurations/cert-manager-kubeaddons-webhook
    ```

1.  For <strong>all</strong> clusters, upgrade capi-components:

    ```bash
    dkp upgrade capi-components
    ```

> 1.  If your cluster was upgraded to 2.1 from 1.8, remove the old cert-manager you marked in step 1:

Reviewer: ```suggestion
1.  If your cluster was upgraded to 2.1 from 1.8, remove the remaining old cert-manager resources from 1.8:
```
